src/Controls/src/Core/Handlers/Items/iOS/CarouselTemplatedCell.cs

```diff
@@ -44,5 +44,10 @@ protected override bool AttributesConsistentWithConstrainedDimension(UICollectio
 		{
 			return false;
 		}
+
+		public override void PrepareForReuse()
```

Comment:
This is reverting changes and fixing the issue or just include the reverts?. Could we include a comment here to avoid remove it by mistake?

Reply:
this fixes the issue, but I still investigating a better way.